external/build-external.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview Builds all external sources. This clears the data/ folder here
+ * and allows all scripts in build/ to write new files there.
+ *
+ * This is intended for use by Cloud Build, or by site devs doing local work.
+ */
+
+// nb. This is important; we load dotenv here and pass environment down.
```

Comment:
This is too cryptic for me to comprehend, Sam. Can you expand the comment?

Reply:
Is there a reason why individual build scripts can't import `.env`, like if we ever want to run one file?

Reply:
I've removed this behavior completely and instead made the child processes run with a cwd of the project's root, so `dotenv` will just work. I think it's probably clearer.